packages/integration-tests/src/entities.ts

```diff
@@ -0,0 +1,77 @@
+export enum Buffer {
+  greeterL2 = "./buffer/greeterL2.txt",
+  executeGreeterTx = "./buffer/executeGreeterTx.txt",
+  NFTtoL1 = "./buffer/NFTtoL1.txt",
+  NFTtoL2 = "./buffer/NFTtoL2.txt",
+  L1 = "./buffer/L1.txt",
+  L2 = "./buffer/L2.txt",
+  L2deposited = "./buffer/L2deposited.txt",
+  paymaster = "./buffer/paymaster.txt",
+  paymasterDeployTx = "./buffer/paymasterDeployTx.txt",
+  paymasterTx = "./buffer/paymasterTx.txt",
+  addressMultiTransferETH = "./buffer/multiTransferETH.txt",
+  txMultiTransferETH = "./buffer/txMultiTransferETH.txt",
+  txMultiTransferCustomTokenI = "./buffer/txMultiTransferCustomTokenI.txt",
+  txMultiTransferCustomTokenII = "./buffer/txMultiTransferCustomTokenII.txt",
+  addressMultiCallMiddle = "./buffer/multiCallMiddle.txt",
+  addressMultiCallCaller = "./buffer/multiCallCaller.txt",
+  addressMultiCallRoot = "./buffer/multiCallRoot.txt",
+  txMultiCallMiddle = "./buffer/txMultiCallMiddle.txt",
+  txMultiCallCaller = "./buffer/txMultiCallCaller.txt",
+  txMultiCallRoot = "./buffer/txmultiCallRoot.txt",
+  txUseMultiCallContracts = "./buffer/txUseMultiCallContracts.txt",
+  txEthTransfer = "./buffer/txEthTransfer.txt",
+  txERC20Withdraw = "./buffer/txERC20Withdraw.txt",
+  txERC20WithdrawOtherAddress = "./buffer/txERC20WithdrawOtherAddress.txt",
+  txEthWithdraw = "./buffer/txEthWithdraw.txt",
+  txEthWithdrawOtherAddress = "./buffer/txEthWithdrawOtherAddress.txt",
+  txERC20Transfer = "./buffer/txERC20Transfer.txt",
+  emptyWalletPrivateKey = "./buffer/emptyWalletPrivateKey.txt",
+  emptyWalletAddress = "./buffer/emptyWalletAddress.txt",
+  failedState = "./buffer/failedState.txt",
+  customToken = "./buffer/customToken.txt",
+}
+
+export enum Logger {
```

Comment:
Why is it called logger? I would split it into `TransferType` enum containing "DEPOSIT", "WITHDRAW" and "TRANSFER". And the rest would just move to constants.

Reply:
@abilevych, I believe we have such name on purpose to differentiate names between `TransactionsType` data , is not it? Should I rename it?